app.rb

```diff
@@ -1,9 +1,35 @@
 require 'sinatra/base'
+require 'sinatra/reloader' # start auto reload
+require_relative './lib/chittermanager' 
 
 class Chitter < Sinatra::Base
+  register Sinatra::Reloader 
+
```

Comment:
You can probably remove things like the `/test` route now that they're no longer needed.